src/components/Footer.tsx

```diff
@@ -1,26 +1,117 @@
+/* eslint-disable react/no-unescaped-entities */
 import React from 'react';
 import { Link } from 'react-router-dom';
-import {
-  createStyles,
-  Theme,
-  WithStyles,
-  withStyles,
-} from '@material-ui/core/styles';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import { PlaceData } from 'common/types';
 
 import { Typography } from 'libs/mui';
 import { HELPFULPLACES_WEBSITE } from 'common/constants';
 
-function Footer({ classes }: WithStyles) {
+interface Props {
+  place: PlaceData;
+  classes: any;
+}
+function Footer({ place, classes }: Props) {
   return (
     <div className={classes.root}>
-      <Typography className={classes.text}>
-        This is an open-source prototype from the Digital Transparency in the
-        Public Realm project. We may share feedback on this page with Helpful
-        Places.
-      </Typography>
-      <Link to={HELPFULPLACES_WEBSITE}>
-        <Typography className={classes.link}>LEARN MORE</Typography>
-      </Link>
+      {place.accountablePrivacyPolicy.length === 0 || null ? (
```

Comment:
Not clear on what this code is doing, what's the `|| null` for? it's returning null if the condition isn't met (so if length isn't 0, it's `null ? condition met : else`